validator/errorbundler.py

```diff
@@ -384,20 +389,28 @@ def print_summary(self, verbose=False, no_color=False):
                                     message=notice,
                                     verbose=verbose)
 
-        if "is_jetpack" in self.metadata and verbose:
-            self.handler.write("\n")
-            self.handler.write("<<GREEN>>Jetpack add-on detected.<<NORMAL>>\n"
-                               "Identified files:")
-            if "jetpack_identified_files" in self.metadata:
-                for filename, data in \
-                        self.metadata["jetpack_identified_files"].items():
-                    self.handler.write((" %s\n" % filename) +
-                                       ("  %s : %s" % data))
-
-            if "jetpack_unknown_files" in self.metadata:
-                self.handler.write("Unknown files:")
-                for filename in self.metadata["jetpack_unknown_files"]:
-                    self.handler.write(" %s" % filename)
+        if verbose:
+            if "framework" in self.metadata:
+                framework = self.metadata["framework"]
+
+                self.handler.write("\n")
+                self.handler.write("<<GREEN>>%s %s framework add-on detected."
+                                   "<<NORMAL>>\n" % (framework["name"],
+                                                     framework["version"]))
+
+            if self.metadata.get("framework_files"):
+                self.handler.write("Framework files:")
+                for filenamename in self.metadata["framework_files"]:
+                    self.handler.write(" %s" % filenamename)
```

Comment:
It'd need to be something like `"\n ".join(...)`, but I'm not sure of all of the implications of writing output that way with this code. I was mainly just updating the current code to use the new data structures.